docs/content/references/ide/move.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Move Analyzer VSCode Extension
+description: The Move extension provides language support features like code navigation, completion, and diagnostics for Move source code.
+---
+The Move Analyzer extension for Visual Studio Code provides language support features for the Move programming language. It enables syntax highlighting, code completion, and advanced features like definition linking and type checking.
+
+## Installing
+
+You can install the Move extension from the Visual Studio Code Marketplace:
+
+- Open VS Code
+- Open the Extensions view (<kbd>⇧</kbd> + <kbd>⌘</kbd> + <kbd>X</kbd> on macOS, <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd> on Windows/Linux) 
+- Search for `mysten.move`
+- Click **Install** on the Move extension by Mysten Labs
+
+You can also use <kbd>Ctrl</kbd> + <kbd>P</kbd> or <kbd>⌘</kbd> + <kbd>P</kbd> and type `ext install mysten.move` to install the extension, or install via the command line: 
+
+```bash
+code --install-extension mysten.move
+```
+
+Installing the Move extension also installs the following extensions:
+
+- [Move Syntax](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax)
+- [Move Trace Debugger](https://marketplace.visualstudio.com/items?itemName=mysten.move-trace-debug)
+
+### Installing `move-analyzer`
+
+The Move extension attempts to install the appropriate `move-analyzer` binary for your platform. If this doesn't work, or you prefer to install it manually, build it with Cargo:
+
+```bash
+cargo install --git https://github.com/MystenLabs/sui.git sui-move-lsp
+```
+By default, the Move extension expects to find the `move-analyzer` binary in `~/.sui/bin`. You can either copy the binary here, or configure the extension to use a different path. 
```

Comment:
Do we need to worry about specifying the location for windows users? I'm guessing not since the only way they can install it is with choco I'm guessing?

Reply:
This is also the path on Windows, just interpolate `~` with %HOMEDIR% or whatever it is. By default this is at `C:\Users\%USERNAME%\.sui\bin`, but that could change based on your home drive.